pyproject.toml

```diff
@@ -0,0 +1,36 @@
+[build-system]
+requires = ["setuptools", "setuptools-scm"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "chemnlp"
+description = "Open source Chemistry dataset & LLM"
+readme = "README.md"
+requires-python = "==3.8.*"  # required for gpt-neox
+dependencies = [
+        "datasets>=2.8.0",
+        "numpy>=1.21.2",
+        "openpyxl>=3.0.9",
+        "pandas>=1.3.3",
+        "peft",
+        "pydantic",
+        "pytdc>=0.3.9",
+        "transformers",
+]
```

Comment:
shall we then also remove the `requirements.txt` and `dev-requirements.txt` and move the latter simply into `extras_require`?

Reply:
Ah, I saw that you already moved them down there. :)

Reply:
But `dev-requirements.txt` seems to be still around?

Reply:
I used to deal with this with `extras` and the mode would be one item in `extras_require`

Reply:
I'm afraid I'm a bit confused as to the change you're suggesting 😳 , I also thought`extras_requires` was a `setup.py` thing?

Reply:
Nice, sure - that's in line with my suggestion below 👇 after these comments. 👌